app/components/CartTile/CartTile.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import _ from 'lodash';
+import './_CartTile.scss';
+import MovieStore from '../../stores/MovieStore';
+import Rating from '../Rating/Rating';
+
+
+const findMovieFromAll = (props) => {
+	let title = props.cartItem.params.title;
```

Comment:
Since you're only using the title in this method, pass that in, rather than the entire props object